js_modules/dagster-ui/packages/ui-core/src/assets/AssetPartitionDetail.tsx

```diff
@@ -100,6 +122,14 @@ export const AssetPartitionDetailLoader = (props: {assetKey: AssetKey; partition
     return <AssetPartitionDetailEmpty partitionKey={props.partitionKey} />;
   }
 
+  const assetFailed = latestRunForPartition?.id
+    ? fetchFailureEvents(latestRunForPartition.id, props.assetKey)
```

Comment:
unfortunately, I don't think this comparison is doing what you want it to do. The way `ASSET_MATERIALIZATION_FAILURE` works is that it is emitted for every asset that is planned to materialize in a run, but doens't get materialized. So this would include assets that are SKIPPED. Eventually, we want to have an additional value on the `ASSET_MATERIALIZATION_FAILURE` event that distinguishes an asset that didn't materialize because of a step failure and one that got skipped, but that is not in place at the moment.